src/net/i2p/crypto/eddsa/spec/EdDSANamedCurveTable.java

```diff
@@ -37,7 +37,7 @@
             Utils.hexToBytes("a3785913ca4deb75abd841414d0a700098e879777940c78c73fe6f2bee6c0352"), // d
             ed25519field.fromByteArray(Utils.hexToBytes("b0a00e4a271beec478e42fad0618432fa7d7fb3d99004d2b0bdfc14f8024832b"))); // I
 
-    private static final EdDSANamedCurveSpec ed25519 = new EdDSANamedCurveSpec(
+    public static final EdDSANamedCurveSpec ED_25519_CURVE_SPEC = new EdDSANamedCurveSpec(
```

Comment:
I guess everything is immutable in `EdDSANamedCurveSpec`, right? 
I can't see a non final property or a hidden internal data structure that would break immutability, so agreed this is probably fine.